packages/pubsub/__tests__/PubSub-unit-test.ts

```diff
@@ -72,6 +72,18 @@ describe('PubSub', () => {
 
             await pubsub.publish('topicA', 'my message');
         });
+
+        test('subscriber is matching MQTT topic wildcards', () => {
+            expect.assertions(5);
+
+            const publishTopic = 'topic/A/B/C';
+            const awsIotProvider = new AWSIoTProvider();
+            expect(awsIotProvider.mqttTopicMatch('topic/A/B/C', publishTopic)).toBe(true);
```

Comment:
```suggestion
            expect(mqttTopicMatch('topic/A/B/C', publishTopic)).toBe(true);
```
After exporting `mqttTopicMatch` function from `MqttOverWSProvider.js`

You will need to import the function and then just  use it on all expect